src/menu_data_helpers.py

```diff
@@ -1,17 +1,45 @@
 CITIES = ('SF', 'CHI', 'NYC', 'SEA',)
 MEALS = ('Breakfast', 'Lunch', 'Dinner', 'Back 2 Basics', 'Happy Hour',)
 
+
+class Allergens:
```

Comment:
You probably want this as a named tuple since you don't have any methods for it, or you want to define all allergens as a parameter for your constructor and assign a default value to it. 
```
Allergens = namedtuple('Allergens', [
    'is_vegetarian',
    'is_vegan',
    'is_gluten_free',
    'is_made_with_dairy',
    'is_made_with_nuts',
    'is_made_with_eggs',
    'is_made_with_soy',])
```

or

```
class Allergens:
    def __init__(self,       
                 is_vegetarian=False, 
                 is_vegan=False, 
                 is_gluten_free=True, 
                 is_made_with_dairy=False, 
                 is_made_with_nuts=False, 
                 is_made_with_eggs=False, 
                 is_made_with_soy=False):
        self.is_vegetarian = is_vegetarian
        self.is_vegan = is_vegan
        self.is_gluten_free = is_gluten_free
        self.is_made_with_dairy = is_made_with_dairy
        self.is_made_with_nuts = is_made_with_nuts
        self.is_made_with_eggs = is_made_with_eggs
        self.is_made_with_soy = is_made_with_soy
```

Then you can refactor the parsing method to be much more readable.